scripts/zones/Halvung/Zone.lua

```diff
@@ -1,10 +1,14 @@
 -----------------------------------
 -- Zone: Halvung (62)
 -----------------------------------
+local ID = zones[xi.zone.HALVUNG]
+-----------------------------------
 local zoneObject = {}
 
 zoneObject.onInitialize = function(zone)
     xi.helm.initZone(zone, xi.helm.type.MINING)
+
+    GetMobByID(ID.mob.COPPER_BORER):setRespawnTime(7200) -- 2 hours
```

Comment:
if he's every 2hrs with no variation and no other conditions, just set him up as timed spawn in its mob group instead of scripted spawn. If there is variation but still "just timed" it should have a similar line in onMobDespawn in its script.